bookbotproject/goodReadsApi.py

```diff
@@ -0,0 +1,23 @@
+from goodreads import client
+import telegrambot
+
+goodreads_key = "UfEAj6yQFcghrsqrZpQ"
+goodreads_secret = "AgAug5etGSAYceT2SeLGBZLMm803I6LiQSxSnGubOjg"
+gc = client.GoodreadsClient(goodreads_key,goodreads_secret)
+
+
+""" this is a draft code, telegram api, goodreads api and wit ai can send messages among themselves or can make necessary things, for the sake of simplicity we created this py file"""
+
+"""TODO : add bunch of methods like ,search(response): takes response as parameter coming from wit.ai intent and searches .... """
```

Comment:
I you have any specific function in mind (for example searchBooks(keyword)) please create the function and put the TODO inside the function. This way people can use the stub function in other places and it will be working fine after we satisfy the TODO.